flink-ml-examples/src/main/java/org/apache/flink/ml/examples/feature/BinarizerExample.java

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
> The public API, i.e., is any changed class annotated with @public(Evolving): (no)
> Does this pull request introduce a new feature? (no)

It would be more proper to change the answer of these two questions to yes, as a new algorithm is a new feature and public API.